content/articles/rebuilding-compiled.md

```diff
@@ -0,0 +1,116 @@
+---
+author_name: "Zarina Mustapha"
+author_url: http://ctl.columbia.edu/about/team/mustapha/
+date: 2016-05-19
+lede: "We moved CompilED to Hugo, a general-purpose static site generator, to test this framework's flexibility and durability to accommodate our blog's requirements and workflow. This post outlines how we converted CompilED from Movable Type, optimized it for accessibility, responsive design, and searchability, implemented lunr.js for search, and streamlined user experience."
+poster: poster-rebuilding-compiled.jpg
+tags: ["hugo","html5","accessibility","responsive design","structured data","microdata"]
+title: "Rebuilding CompilED"
+topics:
+- Process
+type: post
+---
+
+*"We can rebuild [it]. We have the technology."*  
+— Oscar Goldman, *The Six Million Dollar Man*
+
+Earlier in March we decided to move our previous blog to a different platform, with a fresh design in hope to breathe new life into it, and keep it going painlessly.
```

Comment:
simply imho. I'm finding this first sentence a bit awkward and long. Perhaps break into two sentences?